src/codec_rav1e.c

```diff
@@ -134,10 +134,7 @@ static avifBool rav1eCodecEncodeImage(avifCodec * codec, avifImage * image, avif
     if (alpha) {
         rav1e_frame_fill_plane(rav1eFrame, 0, image->alphaPlane, image->alphaRowBytes * image->height, image->alphaRowBytes, byteWidth);
     } else {
-        uint32_t uvHeight = image->height >> yShift;
-        if (uvHeight < 1) {
-            uvHeight = 1;
-        }
```

Comment:
I think the new code does not need this for loop, assuming image->height is nonzero. In the new code, if image->height is nonzero, then uvHeight is nonzero.